src/cli/one_helper/onezone_helper.rb

```diff
@@ -577,7 +580,23 @@ def set_zone(zone_id, temporary_zone)
             puts "Endpoint changed to \"#{zone['TEMPLATE/ENDPOINT']}\" in " <<
                 "#{ENV['HOME']}/.one/one_endpoint"
         end
-        return 0
+
+        return 0 unless zone['TEMPLATE/ONEFLOW_ENDPOINT']
+
+        # Set ONEFLOW_ENDPOINT
+        if temporary_zone
+            puts "Type: export ONEFLOW_URL=#{zone['TEMPLATE/ONEFLOW_ENDPOINT']}"
+        else
+            File.open(ENV['HOME'] + '/.one/oneflow_endpoint', 'w') do |f|
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

OS command injection occurs when untrusted input is used directly in system commands or shell
operations. This vulnerability allows attackers to execute arbitrary commands on the host system.
To prevent OS command injection, use secure alternatives like use proper input validation with proper
allowed characters. For more information about OS command injection,
visit https://owasp.org/www-community/attacks/Command_Injection